### PR TITLE
cleared old bs switch files

### PR DESCRIPTION
**Pull request by jack-han-115**
Fixed issue #<Mantis issue number>: [CR-307](https://limesurvey.atlassian.net/browse/CR-307)
New feature #<Mantis issue number>:
Related to: [previous PR](https://github.com/LimeSurvey/LimeSurvey/pull/3131)

[CR-307]: https://limesurvey.atlassian.net/browse/CR-307?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: application/views/admin/token/massive_actions/_update.php [hunk 1]
```diff
@@ -421,15 +421,6 @@ class="YesNoDatePicker form-control action_datepickerUpdateHiddenField"
 
 
 <?php App()->getClientScript()->registerScript("Tokens:MassActionUpdateView_Scripts", "
-
-   var bindBSSwitch = function(formGroup){
```
Comment (olleharstedt) [inline application/views/admin/token/massive_actions/_update.php:425]: Not needed?

File: application/views/admin/token/massive_actions/_update.php [hunk 2]
```diff
@@ -473,4 +463,4 @@ class="YesNoDatePicker form-control action_datepickerUpdateHiddenField"
     bindClicksInModal(); 
 });
 
-", LSYii_ClientScript::POS_END); ?>
+", LSYii_ClientScript::POS_END); ?>
```
Comment (olleharstedt) [inline application/views/admin/token/massive_actions/_update.php:466]: Is this line correct? Context seems strange.

**Comment by olleharstedt** [comment]
Please see comments. :) Else looks good. :+1:

File: application/views/admin/token/massive_actions/_update.php (hunk 1 above)
Comment (jack-han-115) [reply application/views/admin/token/massive_actions/_update.php:425]: we don't need.

**Comment by olleharstedt** [comment]
OK to me! Didn't test anything tho. Which Jira task is this connected to again? Please include it in the PR title so it's easy to find. :face_with_spiral_eyes:

**Comment by jack-han-115** [comment]
i add ticket number as prefix of PR title.

**Comment by olleharstedt** [comment]
Oh yeah, the branch name, great. :+1:

**Comment by olleharstedt** [comment]
There was a tiny warning regarding PHP style here when you merged @jack-han-115 . I fixed it already, so no worries, but better to be fixed before merge in the future. :) 

```
FILE: ...k/LimeSurvey/LimeSurvey/application/controllers/admin/Tokens.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 2760 | ERROR | [x] The first expression of a multi-line control
      |       |     structure must be on the line after the opening
      |       |     parenthesis
 2761 | ERROR | [x] The closing parenthesis of a multi-line control
      |       |     structure must be on the line after the last
      |       |     expression

**Comment by jack-han-115** [comment]
ok, thank you.

**Pull request merged**